jekyll/_cci2/circleci-config-sdk.adoc

```diff
@@ -0,0 +1,61 @@
+---
+version:
+- Cloud
+---
+= CircleCI Config SDK
+:page-layout: classic-docs
+:page-liquid:
+:page-description: The CircleCI Config SDK is a JavaScript package library for generating CircleCI config YAML.
+:icons: font
+:toc: macro
+:toc-title:
+
+[#overview]
+== Overview
+
+The link:https://circleci-public.github.io/circleci-config-sdk-ts[CircleCI Config SDK] is a JavaScript package library (also compatible with TypeScript) for generating a CircleCI YAML configuration file. Use the SDK to:
+
+* Replace writing YAML conguration files
+* Augment YAML configuration files
```

Comment:
I think this one might be true if it were included under `Enhance CircleCI's <<using-dynamic-configuration#,dynamic configuration>>`

I'm thinking for an alternative but I think we should remove this line